src/components/Me/Navigation/Navbar.css

```diff
@@ -0,0 +1,62 @@
+html {
```

Comment:
This is affecting the main app so it should probably need to be handled in a different way.